### DISCUSSION TITLE
Attribute not known: "@db.Uuid".

### DISCUSSION THREAD

**Discussion by SebastianLJ** [prisma/prisma#9762] [Q&A] [answered]
I get this error im my schema file, in VSC
<img width="929" alt="Screenshot 2021-10-13 at 18 22 26" src="https://user-images.githubusercontent.com/37780699/137174386-2957c762-143f-43df-b3f4-defe5d5769e9.png">

**Comment by janpio** [comment, marked as answer]
What output do you get when you run `prisma format` with this schema?
How did you get this schema - write it by hand or via `prisma db pull`?

**Reply by SebastianLJ** [reply]
The schema works fine in implementation and the error is just a visual one in VSC I can't get rid of. The schema is written by hand

**Reply by janpio** [reply]
Could it be that you have a _really_ old version of the VSCode extension installed then? Both the underlined features were not present from the beginning, so if you are using from before these existed, it would make sense that the extension does not know about them.

**Reply by SebastianLJ** [reply]
Im using the newest version of the Prisma extension. It's only a problem on my local environment and it does not occur for other project members...

**Reply by janpio** [reply]
What version is that? Does this happen in all projects or only in this one?

**Reply by SebastianLJ** [reply]
I use 0.0.32 and it happens for all projects

**Reply by janpio** [reply]
The newest version of the extension is 3.3.2: https://marketplace.visualstudio.com/items?itemName=Prisma.prisma

**Reply by SebastianLJ** [reply]
my VSC was outdated, an update fixed this... Thanks for the time @janpio

**Reply by janpio** [reply]
Happy to help, good to know we could figure this out together 👍

**Comment by VitorM3** [comment]
Sorry if my english is not good, i'm using google translator.
I'm having the same problem but the update didn't fix it, what can I do?

**Reply by janpio** [reply]
Are you using the newest VS Code extension version, which right now is `3.12.0`?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
